plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/dev/Logging.js

```diff
@@ -103,13 +107,39 @@ Ext.define('NX.controller.dev.Logging', {
           }
         },
 
-        'nx-dev-logging button[action=dump]': {
-          click: function (button) {
-            //encode selected rows as JSON and dump to log, intended to facilitate sharing log info (i.e. bug reports)
-            me.logDebug(Ext.encode(Ext.Array.pluck(this.getPanel().getSelectionModel().getSelection(), 'data')));
-          }
+        'nx-dev-logging button[action=export]': {
+          click: me.exportSelection
         }
       }
     });
+  },
+
+  /**
+   * Export selected rows to window.
+   *
+   * @private
+   */
+  exportSelection: function() {
+    var win,
+        doc,
+        selected;
+
+    win = NX.Windows.open('', '', 'width=640,height=480');
```

Comment:
works fine, and is definitely an improvement, but why a native window and why raw html instead of ExtJS components?